src/plugins/crypto/gcrypt_operations.c

```diff
@@ -0,0 +1,221 @@
+/***************************************************************************
+ gcrypt_operations.c  -  gcrypt cryptography interface
+ -------------------
+ begin                : Mon Aug 31 18:04:14 CEST 2015
+ copyright            : (C) 2015 by Peter Nirschl
+ email                : peter.nirschl@gmail.com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the BSD License (revised).                      *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "gcrypt_operations.h"
+#include <gcrypt.h>
+
+static gcry_cipher_hd_t gcry_handle;
+static gcry_error_t gcry_err;
+
+
+void elektraCryptoGcryClearKeyIv()
```

Comment:
Why encode Gcry into the name? Wouldn't it be nice if another crypto API can be used below?

   int elektraCryptoGcryEncrypt(Key *k)

seems to be highlevel enough that it should work.